### PR TITLE
Fix typos

### PR DESCRIPTION
**Pull request by szepeviktor**
Found a few misspellings.

Are these all vendored files?
    "admin/js/jspreadsheet.js"
    "admin/js/jsuites.js"
    "js/*.min.js"
Excluded from checking!

### CODE REVIEW

File: libraries/simplexlsx.class.php [hunk 1]
```diff
@@ -268,7 +268,7 @@ protected function _unzip($filename, $is_data = false)
 /*            } else {
 				switch ($aP['CM']) {
 					case 0: // Stored
-						// Here is nothing to do, the file ist flat.
+						// Here is nothing to do, the file is flat.
```
Comment (szepeviktor) [inline libraries/simplexlsx.class.php:271]: 🇩🇪 Das File ist flach.

**Comment by TobiasBg** [comment]
Hi @szepeviktor,

thanks a lot for this! Great finds!
I have merged this in https://github.com/TablePress/TablePress/commit/7f4ec0a89325738ae95135c43181eabf327e6567

I have also removed your changes in `libraries/simplexlsx.class.php` as that is an external library as (in addition to the JS files that you mention). Technically, `libraries/evalmath.class.php` is also external, but it doesn't seem to be getting updates anymore, so that I have "adopted" it so to speak.

Best wishes,
Tobias

**Comment by szepeviktor** [comment]
Glad to contribute.

You could use `typos` in every project of yours.

**Comment by TobiasBg** [comment]
Do you mean this project: https://github.com/crate-ci/typos?
That does indeed look interesting, thanks!

**Comment by szepeviktor** [comment]
Yes.

**Pull request closed without merging**